src/utils/parseImageUrl.ts

```diff
@@ -1,6 +1,6 @@
 const parseImageUrl = (url: string) => {
   if (process.env.NODE_ENV === "development") {
-    return process.env.STRAPI_URL + url
+    return `${process.env.STRAPI_URL + url}?format=webp`
   }
   return url
```

Comment:
If we end up implementing Tietokilta/tikweb-cms#51 or similar in production, we should also add `?format=webp` here.

If we do it that way, the image URL will also move to `https://<strapi>/uploads`, which will require adding `STRAPI_URL` to each.